src/modules/pages/NewApiPage/NewApiForm.js

```diff
@@ -55,6 +57,7 @@ const ApiForm = props => {
         label: item,
         value: item,
     }));
+    const checkOnPattern = pattern => value => value && !validation(pattern)(value) ? true : undefined;
```

Comment:
Perhaps we could extract this function to somewhere else for reusing
pattern-check.js
```js
export default pattern => value => value && !validation(pattern)(value) ? true : undefined;
```